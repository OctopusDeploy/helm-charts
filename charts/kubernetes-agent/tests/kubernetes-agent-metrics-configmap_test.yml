suite: "tentacle configuration"
templates:
  - templates/tentacle-configmap.yaml
tests:
  - it: "should match snapshot"
    asserts:
      - matchSnapshot: {}
      
  - it: "defaults to empty data (no data field)"
    asserts:
      - notExists:
          path: data
