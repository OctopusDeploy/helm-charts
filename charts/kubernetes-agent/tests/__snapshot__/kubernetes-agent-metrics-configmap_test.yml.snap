should match snapshot:
  1: |
    apiVersion: v1
    kind: ConfigMap
    metadata:
      name: kubernetes-agent-metrics
      namespace: NAMESPACE
